crates/thetawave_interface/src/spawnable.rs

```diff
@@ -121,11 +129,12 @@ pub enum ItemType {
 }
 
 /// Type that encompasses all spawnable effects
-#[derive(Deserialize, Debug, Hash, PartialEq, Eq, Clone, Display)]
+#[derive(Deserialize, Debug, Hash, PartialEq, Eq, Clone, Display, Default)]
```

Comment:
Im tempted to say let's not have a default here since there is no "canonical" effect type. But it's...fine